addon/helpers/d3-tick-format.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+import { timeFormat } from 'd3-time-format';
+
+export function d3TickFormat([tickFormat]) {
```

Comment:
@ivanvanderbyl alright - here's an example of how we are thinking of using this helper:
https://github.com/LocusEnergy/ember-sparkles/pull/4/files#diff-33855cd461c3ef518666bb5cdea92472R9

are you proposing that we keep this file (`d3-tick-format`) but just rename it? The idea is that the axis helper needs to either take a string or a function for it's `tickFormat` argument